docs/guides/server/configuration.adoc

```diff
@@ -101,6 +107,23 @@ You can also specify an explicit configuration file location using the `[-cf|--c
 
 <@kc.start rootParameters="--config-file=/path/to/myconfig.conf"/>
 
+=== Format for loading properties from a Java KeyStore
```

Comment:
I think we should make this section to be more explicit about `Setting sensitive options using a Java KeyStore` or something like that.